### PR TITLE
Section 3 comments

### PR DESCRIPTION
**Pull request by lachellel**
52 comments reviewed. 
Updates (accepted) or no changes for 47 comments. 

Five outstanding items not addressed and for discussion: 

1. CAA records
2. High risk certificate requests 
3.  Clearer definition of Subscriber 
4.  Uniqueness for subscriber common names
5.  Addition of domain validation method: test certificates

### CODE REVIEW

**Comment by lachellel** [comment]
closes #272 closes #273 closes #274 closes #275 closes #276 closes #278 closes #282 closes #337 closes #339 closes #340 closes #341 closes #466 closes #342 closes #343 closes #344 closes #345 closes #346 closes #347 closes #348 closes #350 closes #351 closes #214 closes #215 closes #216

File: certificate-policy.md [hunk 1]
```diff
@@ -303,57 +288,60 @@ This Certificate Policy allows for procedures adhering to:
...
 
-The CA shall confirm that, as of the date the Certificate issues, the CA has validated each Fully-Qualified Domain Name (FQDN) listed in the Certificate using at least one of the methods listed in Section 3.2.2.4.x.
+Issuing CAs shall confirm that, as of the date the Certificate issues, the CA has validated each Fully-Qualified Domain Name (FQDN) listed in the Certificate using at least one of the methods listed in Section 3.2.2.4.x.
 
-Completed confirmations of Applicant authority may be valid for the issuance of multiple certificates over time. In all cases, the confirmation shall have been initiated within the time period specified in the relevant requirement (Section 3.3.1 of this document) prior to certificate issuance. For purposes of domain validation, the term Applicant includes the Applicant's U.S. Government Department, Agency, Commission, component, or other organizational unit defined in United States Code.
+Issuing CAs shall maintain a record of which domain validation method, including the relevant Baseline Requirements version number, that were used to validate every domain.
```
Comment (konklone) [inline certificate-policy.md]: Grammar is weird here. How about (changes highlighted):

> Issuing CAs shall maintain a record of which domain validation method, including the relevant Baseline Requirements version number, _**was**_ used to validate _**each**_ domain.

File: certificate-policy.md [hunk 2]
```diff
@@ -303,57 +288,60 @@ This Certificate Policy allows for procedures adhering to:
...
+This validation method defined by the Baseline Requirements is not allowed under this CP.
 
-##### 3.2.2.4.9 [Reserved]
+##### 3.2.2.4.9 Test Certificate
+This validation method defined by the Baseline Requirements is not allowed under this CP.
 
 ##### 3.2.2.4.10. TLS Using a Random Number
-
-Confirming the Applicants control over the requested FQDN by confirming the presence of a Random Value within a Certificate on the Authorization Domain Name which is accessible by the CA via TLS over an Authorized Port.
+This validation method confirms the Applicants control over the requested FQDN by confirming the presence of a Random Value within a Certificate on the Authorization Domain Name which is accessible by the CA via TLS over an Authorized Port.
```
Comment (konklone) [inline certificate-policy.md]: Typo: `Applicants` -> `Applicant's`

**Review by konklone — APPROVED** [review]
Noted two tiny errors. I'll edit on-branch and merge directly.

**Comment by konklone** [comment]
Made two typo fixes and merged. :+1:

**Pull request merged**